### ISSUE TITLE
Назови файлы хорошо, чтобы я понимал что к какому заданию относится

### ISSUE THREAD

**Issue by antosha417** [leraved/prog_Vedernikova#3]
Как вариант можно раскидать по папкам с  названиями из таблицы

**Comment by leraved** [comment]
переименовала файл в picture_ball

**Comment by antosha417** [comment]
гитхаб это не помойка куда ты скидываешь всё что под руку попадёт. Наведи порядок, чтобы было просто разобраться

**Comment by leraved** [comment]
Пойдет?

**Comment by antosha417** [comment]
Да, так лучше. Писал уже в телеграмме, не нужно куча файлов ``picture.py`` . Достаточно одного. Сделала, закомитила. Сделала лучше. Опять коммит. Ещё лучше - снова коммит. Добавила анимацию - коммит. И будет 1 файл, в котором будет видна история разработки. И в такой процесс легко добавлять какие-нибудь новые штуки, хоть 50. Не нужно будет 50 разных файлов.

**Comment by leraved** [comment]
Вроде бы нужно было 3 отдельных программы, в каждую из которых добавляется усовершенствование. Нет?

**Comment by antosha417** [comment]
Ты не согласна, что в итоге ты усовершенствуешь одну и ту же программу?

**Comment by leraved** [comment]
ок. я переделаю

**Comment by antosha417** [comment]
Если не согласна, оставь так

**Comment by antosha417** [comment]
Я не заставляю , просто говорю своё видение
